src/app/org/schedule/page.tsx

```
@@ -58,6 +58,24 @@ const axisLabels: Record<AxisType, string> = {
+  "America/New_York",
+  "America/Phoenix",
+  "Pacific/Honolulu",
+  "America/Anchorage",
+  "UTC",
+] as const;
+const supportedTimeZones = (
+  Intl as typeof Intl & { supportedValuesOf?: (key: "timeZone") => string[] }
+).supportedValuesOf?.("timeZone");
+const ORGANIZATION_TIME_ZONES = Array.from(
+  new Set([...(supportedTimeZones ?? FALLBACK_ORGANIZATION_TIME_ZONES), "UTC"]),
+).sort();
```

ESLint

[error] Use spread syntax [...iterable] instead of Array.from(iterable) when no mapper function is provided

(e18e/prefer-spread-syntax)